pkg/logbridge/logbridge.go

```diff
@@ -100,8 +129,9 @@ func (lb *LogBridge) lossyCopy(r io.Reader, lines chan []byte) {
 		case lines <- line:
 		default:
 			droppedLines++
+			lb.droppedLineCount.Inc(1)
```

Comment:
Isn't this limit just based on the channel capacity and not the lovely rate limiter you've just added?